### DISCUSSION TITLE
How I can publish an article for Vercel Blog (Community)?

### DISCUSSION THREAD

**Discussion by marcelo-earth** [vercel/community#1728] [Idea]
### Summary

Hi everyone! I just see that in Vercel Blog there is a section for Community articles.

Is this open to submissions? I would like to share an article about a web application that I made with Next.js 13.2.

Thanks in advance! 😄.

**Comment by amyegan** [comment]
That's awesome, Marcelo! It's a good idea to host it on your own site so you can keep control over it for future updates, etc. But I would be happy to feature it in the [weekly news post](https://github.com/orgs/vercel/discussions?discussions_q=news+cache+sort%3Adate_created) if you share a link once it's published 😃

**Reply by marcelo-earth** [reply]
I understand. Thank you so much @amyegan! 😁
